lib/src/store.dart

```diff
@@ -146,4 +147,24 @@ class Store extends Stream<Store> with Disposable {
       trigger();
     }));
   }
+
+  /// A convenience method for listening to an [action] and triggering
+  /// automatically once the callback for said action has completed.
```

Comment:
I'm not sure that we actually await the callback. If it's `void` then we can just call it, but if it were to return a future I'm not sure...

Reply:
it is awaited, but nothing done with the return value.